config/locales/shared/en.yml

```diff
@@ -10,6 +10,7 @@ en:
       landmark_label: Official government website
       lock: Lock
       lock_description: A locked padlock
+      ie_not_supported: Internet Explorer 11 will not be supported after %{date}
```

Comment:
Since we're not rendering this in the banner anymore, would another place be more suitable for this string? Maybe `account.login.ie_not_supported` ?